modules/tinymce/src/plugins/table/main/ts/core/Util.ts

```diff
@@ -8,6 +8,7 @@
 import { Arr, Optional, Strings } from '@ephox/katamari';
 import { TableLookup } from '@ephox/snooker';
 import { Attribute, Compare, SugarElement } from '@ephox/sugar';
+
 import Editor from 'tinymce/core/api/Editor';
```

Comment:
Just noticed that for some reason we don't have a lint rule enforcing that there should be a blank line between imports and the start of actual code. We should (separately) fix that

Reply:
Yeah, we should. I've fixed this specific case for now. If anyone wants to try this, it's a separate linter rule: https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/newline-after-import.md